### ISSUE TITLE
[question] what is padding?

### ISSUE THREAD

**Issue by carnoxen** [cezane/optee_rsa_example#2]
```c
#define RSA_KEY_SIZE 1024
#define MAX_PLAIN_LEN_1024 86 // (1024/8) - 42 (padding)
#define RSA_CIPHER_LEN_1024 (RSA_KEY_SIZE / 8)
```
@cezane, 
In your header file, there are macros with one comments. Can I find the padding in the [official specification](https://globalplatform.org/wp-content/uploads/2018/06/GPD_TEE_Internal_Core_API_Specification_v1.1.2.50_PublicReview.pdf)?

**Comment by carnoxen** [comment]
```c
#define RSA_KEY_SIZE 1024
#define MAX_PLAIN_LEN_1024 86 // (1024/8) - 42 (padding)
#define RSA_CIPHER_LEN_1024 (RSA_KEY_SIZE / 8)
```

In your header file, there are macros with one comments. Can I find the padding in the [official specification](https://globalplatform.org/wp-content/uploads/2018/06/GPD_TEE_Internal_Core_API_Specification_v1.1.2.50_PublicReview.pdf)?

**Comment by carnoxen** [comment]
```c
#define RSA_KEY_SIZE 1024
#define MAX_PLAIN_LEN_1024 86 // (1024/8) - 42 (padding)
#define RSA_CIPHER_LEN_1024 (RSA_KEY_SIZE / 8)
```
@cezane, 
In your header file, there are macros with one comments. Can I find the padding in the [official specification](https://globalplatform.org/wp-content/uploads/2018/06/GPD_TEE_Internal_Core_API_Specification_v1.1.2.50_PublicReview.pdf)?

**Comment by cezane** [comment]
Hello, @carnoxen 

I think the following links will help you with the concept of "padding":
https://en.wikipedia.org/wiki/Padding_(cryptography)
https://cryptography.fandom.com/wiki/Padding_(cryptography)

Regarding the code, I saw that the constant `MAX_PLAIN_LEN_1024` was declared but never used. So, I think I declared it for some reason, but later gave up that idea or figured out that it was not necessary, and just forgot to delete that declaration. 

Regarding the official specification, you will see some references for paddings in the functions. The links above will help you to understand the use of a padding.
